src/Service.Host/client/src/components/parts/mechPartSource/tabs/ProposalTab.js

```diff
@@ -299,6 +312,7 @@ function ProposalTab({
                     fetchItems={searchDepartments}
                     links={false}
                     clearSearch={clearSearchDepartments}
+                    required
```

Comment:
I dunno if this flag really does anything? If projectCode is becoming a mandatory field when creating source sheets, a condition/check in the domain code would be better